<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>75</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Replace with:</string>
       </property>
       <property name="buddy">
        <cstring>replaceCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="replaceCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Find:</string>
       </property>
       <property name="buddy">
        <cstring>findCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="findCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QToolButton" name="replacePreviousButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="replaceNextButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replaceAllButton">
         <property name="text">
          <string>Replace All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="findPreviousButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="findNextButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="matchCaseCheck">
         <property name="text">
          <string>&amp;Case sensitive</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="regExpCheck">
         <property name="text">
          <string>Use regular e&amp;xpressions</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>findCombo</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>findPreviousButton</tabstop>
  <tabstop>replaceCombo</tabstop>
  <tabstop>replaceNextButton</tabstop>
  <tabstop>replacePreviousButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>matchCaseCheck</tabstop>
  <tabstop>regExpCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
